devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -204,7 +204,7 @@ def setLevel(value, rate = null) {
 	def additionalCmds = []
 	if (device.getDataValue("model") == "iQBR30" && value.toInteger() > 0) { // Handle iQ bulb not following spec
 		additionalCmds = zigbee.on()
-	} else if (device.getDataValue("manufacturer") == "MRVL") { // Handle marvel stack not reporting
+	} else if (isMRVL() || isLeviton()) { // Handle marvel stack not reporting
 		additionalCmds = refresh()
 	}
```

Comment:
Doing it this way does mean we are also going to send a read for the on/off status every time level is set, which seems unnecessary.

Reply:
Good point. I've changed it already.